scripts/lsstinstall

```diff
@@ -348,6 +348,18 @@ else
     echo "$EUPS_PKGROOT" > "$EUPS_PATH/pkgroot" || fail "Unable to write pkgroot file"
 fi
 
+# Copy postinstall
+if [ -z "$EUPS_PATH" ]; then
```

Comment:
I'm not sure why this check is failing in the actions because it's clear that $EUPS_PATH does exist since it is used 2 lines above.